common/v2/components/MobileNavBar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+
+import { BREAK_POINTS, COLORS } from '../theme';
+
+const MobileNavBarStyled = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: -19px 0 0;
+
+  > div {
+    &.tab {
+      position: relative;
+      flex-basis: 0;
+      flex-grow: 1;
+      width: 100%;
+      max-width: 100%;
+
+      padding: 23px 0;
```

Comment:
Could you use `SPACING` styling constant ?

Reply:
No problem, I was just using values from designs.